src/Drawer/index.js

```diff
@@ -99,12 +84,13 @@ const Drawer = ({
     >
       <div className={`navigation-container--${position}`}>
         {showClose && (
-          <div
-            className={`navigation-button navigation-button--${position} alignChild--center--center`}
+          <button
+            className={`button--reset navigation-button navigation-button--${position} alignChild--center--center`}
```

Comment:
the humble `button--reset` class does a lot of the annoying button restyling work for us, making it easier to use real `button` elements instead of divs for custom controls like this